scripts/changed-files

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+git diff --name-only --diff-filter=d main
```

Comment:
For this to work in CI, I think we'd have to increase the depth of the checkout action

Reply:
Fixed. I've changed the github action to fetch the HEAD of the main and the work branch